apps/widclkinfo/widget.js

```diff
@@ -47,4 +57,20 @@ if (!require("clock_info").loadCount) { // don't load if a clock_info was alread
       }
     }
   };
-}
\ No newline at end of file
+
+  Bangle.on("hidden", () => {
```

Comment:
Do we want:

```suggestion
  Bangle.on("widgets-hidden", () => {
```

(and similarly below)

Reply:
Oops yes, I didn't copy over correctly